### PR TITLE
fix: Missing APIs

### PR DESCRIPTION
**Pull request by Anush008**
This PR
- Marks the delete vectors API as public.
- Adds the missing [batch points update](https://qdrant.tech/documentation/concepts/points/#batch-update) API.
- Adds support for positive and negative vectors in the recommend APIs.

### CODE REVIEW

File: src/Qdrant.Client/QdrantClient.cs [hunk 1]
```diff
@@ -3135,6 +3175,8 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
 		string groupBy,
 		IReadOnlyList<PointId> positive,
 		IReadOnlyList<PointId>? negative = null,
+		ReadOnlyMemory<Vector>? positive_vectors = null,
```
Comment (IvanPleshkov) [inline src/Qdrant.Client/QdrantClient.cs:3178]: Codestyle `positiveVectors` and `negativeVectors`

**Review by IvanPleshkov — CHANGES_REQUESTED** [review]

File: src/Qdrant.Client/QdrantClient.cs (hunk 1 above)
Comment (Anush008) [reply src/Qdrant.Client/QdrantClient.cs]: https://github.com/qdrant/qdrant-dotnet/pull/42/commits/2141e763ba18f9d237fb68a1ce0bee092834b750

**Review by IvanPleshkov — APPROVED** [review]

File: src/Qdrant.Client/QdrantClient.cs [hunk 2]
```diff
@@ -3163,6 +3205,16 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
...
 			request.Negative.AddRange(negative);
 
+		if (positiveVectors is not null)
+		{
+			Populate(request.PositiveVectors, (ReadOnlyMemory<Vector>)positiveVectors);
+		}
+
+		if (negativeVectors is not null)
+		{
+			Populate(request.PositiveVectors, (ReadOnlyMemory<Vector>)negativeVectors);
```
Comment (russcam) [inline src/Qdrant.Client/QdrantClient.cs:3215]:
```suggestion
			Populate(request.NegativeVectors, (ReadOnlyMemory<Vector>)negativeVectors);
```

Comment (Anush008) [reply src/Qdrant.Client/QdrantClient.cs]: 🦅👀
Thank you.

File: src/Qdrant.Client/QdrantClient.cs [hunk 3]
```diff
@@ -2867,6 +2883,16 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
...
 
+		if (positiveVectors is not null)
+		{
+			Populate(request.PositiveVectors, (ReadOnlyMemory<Vector>)positiveVectors);
+		}
+
+		if (negativeVectors is not null)
+		{
+			Populate(request.NegativeVectors, (ReadOnlyMemory<Vector>)negativeVectors);
+		}
```
Comment (russcam) [inline src/Qdrant.Client/QdrantClient.cs:2894]:
```suggestion
		if (positiveVectors is not null)
			Populate(request.PositiveVectors, positiveVectors.Value);

		if (negativeVectors is not null)
			Populate(request.NegativeVectors, negativeVectors.Value);
```

File: src/Qdrant.Client/QdrantClient.cs [hunk 4]
```diff
@@ -3163,6 +3205,16 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
...
```
Comment (russcam) [inline src/Qdrant.Client/QdrantClient.cs:3216]:
```suggestion
		if (positiveVectors is not null)
			Populate(request.PositiveVectors, positiveVectors.Value);

		if (negativeVectors is not null)
			Populate(request.NegativeVectors, negativeVectors.Value);
```

File: tests/Qdrant.Client.Tests/PointTests.cs [hunk 5]
```diff
@@ -305,6 +305,64 @@ public async Task Count_with_filter()
...
+			positive: new PointId[] { 8 },
+			positiveVectors: new Vector[] {
+				new float[] { 3.5f, 4.5f }
+			},
+			limit: 1
+		);
+
+		var point = Assert.Single(points);
+		Assert.Equal(9ul, point.Id);
+	}
```
Comment (russcam) [inline tests/Qdrant.Client.Tests/PointTests.cs:364]: Move this test method next to the other Recommend tests

**Review by russcam — APPROVED** [review]
LGTM, left suggestions

File: tests/Qdrant.Client.Tests/PointTests.cs (hunk 5 above)
Comment (Anush008) [reply tests/Qdrant.Client.Tests/PointTests.cs]: https://github.com/qdrant/qdrant-dotnet/pull/42/commits/30f944cd5557184d9520cc129d175e7deb94976c

**Pull request merged**